Services/AuthenticationService/Commands/LoginCommand.cs

```diff
@@ -40,7 +40,7 @@ private bool CheckUserCredentials(User user, UserEmailPassword credentials)
             this.busControl = busControl;
         }
 
-        public async Task<string> Execute(UserEmailPassword data)
+        public async Task<UserToken> Execute(UserEmailPassword data)
         {
             User user = await GetUserFromUserService(data);

```

Comment:
Why do we have to return user?

Reply:
To store id-token pair in client's storage

Reply:
Let's think about that. Don't we have to add Id in UserEmailPassword and receive it?  I don't think that it is good idea to use user here